CoinDock-api/app/Http/Controllers/V1/RecoveryKeyController.php

```diff
@@ -44,10 +45,34 @@ public function create(User $user)
                     'completed' => 3,
                 ],
             ],
-            200,
+            Response::HTTP_OK
         );
     }
 
+
+
+    public function reGenerateRecoveryKeys(User $user){
+
```

Comment:
Are we still using reGenerateRecoveryKeys()?

Reply:
Sorry.. forgot to remove it. removed now